src/main/java/org/czekalski/petapiintegration/apiclient/v1/mapper/AnimalDtoToAnimalMapper.java

```diff
@@ -0,0 +1,32 @@
+package org.czekalski.petapiintegration.apiclient.v1.mapper;
+
+import org.czekalski.petapiintegration.apiclient.v1.dto.AnimalDto;
+import org.czekalski.petapiintegration.apiclient.v1.dto.BreedsDto;
+import org.czekalski.petapiintegration.resourceserver.model.Animal;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AnimalDtoToAnimalMapper implements Mapper<AnimalDto, Animal> {
+    @Override
+    public Animal map(AnimalDto animalDto) {
+        if (animalDto == null) {
+            return null;
+        } else {
+            Animal animal = new Animal();
+            animal.setId(animalDto.getId());
+            animal.setName(animalDto.getName());
+            animal.setBreed(this.breedDtoToBreed(animalDto.getBreeds()));
+            animal.setAge(animalDto.getAge());
+            animal.setGender(animalDto.getGender());
+            return animal;
```

Comment:
To można właśnie ogarnąć albo builderem, który wypełni konstruktor albo po prostu konstruktorem

Reply:
Poprawiono: fc8cfdb7a930ad294dac115286764f871917cdee